docs/README.md

```diff
@@ -14,21 +28,7 @@ Otherwise, check out the following guides to using and developing IPFS:
 - [Installing command completion](command-completion.md)
 - [Mounting IPFS with FUSE](fuse.md)
 - [Installing plugins](plugins.md)
-
-
-## API Support
-
-- [How to Implement an API Client](implement-api-bindings.md)
-- [Connecting with Websockets](transports.md) — if you want `js-ipfs` nodes in web browsers to connect to your `go-ipfs` node, you will need to turn on websocket support in your `go-ipfs` node.
-
-
-## Developing `go-ipfs`
-
-- Building on…
-    - [Windows](windows.md)
-- [Performance Debugging Guidelines](debug-guide.md)
-- [Release Checklist](releases.md)
-
+- [Setting up an IPFS Gateway](https://github.com/ipfs/go-ipfs/blob/master/docs/gateway.md)
```

Comment:
@ipfs/docs found these docs inside the go-ipfs repo. Seem that they should make their way into docs.ipfs.io as well, at least at some point.

Reply:
@daviddias See https://github.com/ipfs/docs/pull/303 😊